docs/troubleshoot/app-framework/appserver-cluster.md

```diff
@@ -0,0 +1,24 @@
+# Managing Cluster Mode for app-server
+
+On the Zowe servers, the component "app-server" has an environment variable "ZLUX_NO_CLUSTER" which controls whether or not it uses cluster mode. Cluster mode is enabled by default. However, you might need to disable cluster mode under certain circumstances. When cluster mode is disabled, make sure you are aware of the potential drawbacks and benefit. 
+
+When you **disable** cluster mode, you will lose the following benefits:
+
+1. **Performance under high user Count:** This is due to the absence of redundant workers, which can impact the system's efficiency when dealing with a large number of users.
+
+2. **Reduced downtime during unexpected exceptions:** The low-downtime characteristic, where only one request is interrupted compared to around 15 seconds of downtime, is compromised.
+
+However, this trade-off comes with a specific benefit:
+
+- **Handling Network Interface Failures:** It addresses a limitation in Node.js where the system cannot effectively manage a network interface going offline. This limitation can result in an endless socket read loop until the process is manually terminated.
```

Comment:
I think this is no longer true. At the time I made a ticket describing the issue, nodejs had a bug. A significant amount of time has passed since then, and the bug is fixed.

But, this ZLUX_NO_CLUSTER still exists in case it is needed for some reason.

Reply:
Thanks. This paragraph is removed.